packages/react-components/react-tree/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-tree",
-  "version": "9.0.0-beta.32",
-  "description": "React components for building web experiences",
+  "version": "9.0.0",
```

Comment:
Curious what the first published version will be :-)

Reply:
Yeah, I'm not sure if I should just keep the `9.0.0-beta.32` and do a minor or this approach of pinning it to `9.0.0` and doing no update version with the changefile

cc:  @Hotell ,  @tudorpopams , @marcosmoura

Reply:
Just updated it to do a patch instead 😁